.prod/uwsgi.ini

```diff
@@ -6,6 +6,13 @@ static-map = /static=/var/static
 static-map = /media=/app/var/media
 uid = appuser
 gid = appuser
+
+# Reload workers regularly to keep memory fresh
+# and ease potential memory leaks
+max-requests = 1000                  # Restart workers after this many requests
```

Comment:
What is our default for this? Or the current value? I found a post from the StackOverflow that prefers these (https://stackoverflow.com/a/44641109/784642):

```
Haharakiri = 20 # respawn processes taking more than 20 seconds
limit-as = 256 # limit the project to 256 MB
max-requests = 5000 # respawn processes after serving 5000 requests
daemonize = /var/log/uwsgi/yourproject.log # background the process & log
```